status/backend.json

```diff
@@ -207,7 +207,10 @@
       }
     },
     "ioMarkdown": {
-      "disabledForMessagesAndServices": false
+      "min_app_version": {
+        "ios": "3.1.0.0",
+        "android": "3.1.0.0"
```

Comment:
What do you think to wait before define the next version?

Reply:
All right, set to 0.0.0.0